LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/PackageServices.cs

```diff
@@ -51,6 +51,6 @@ public class PackageServices
         /// Gets or sets the Date Updated
         /// </summary>
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public DateTime? DateUpdated { get; set; }
+        public DateTimeOffset? DateUpdated { get; set; }
```

Comment:
This filed should be Computed not Identity, identity will only receive a value on insert